CMP1903_A1_2324/Game.cs

```diff
@@ -3,19 +3,38 @@
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
+using System.Diagnostics;
 
 namespace CMP1903_A1_2324
 {
-    internal class Game
+    public class Game
     {
-        /*
-         * The Game class should create three die objects, roll them, sum and report the total of the three dice rolls.
-         *
-         * EXTRA: For extra requirements (these aren't required though), the dice rolls could be managed so that the
-         * rolls could be continous, and the totals and other statistics could be summarised for example.
-         */
+        private List<Die> _diceRolls = new List<Die>();
```

Comment:
I like the use of lists here, able to expand the number of dice very easily by using this.